pulp_container/app/utils.py

```diff
@@ -209,4 +209,6 @@ def validate_manifest(content_data, media_type, digest):
         validate(content_data, schema_validator)
     except ValidationError as error:
         # fail on the first encountered error
-        raise ManifestInvalid(reason=f'{".".join(error.path)}: {error.message}', digest=digest)
+        raise ManifestInvalid(
+            reason=f'{".".join(map(str, error.path))}: {error.message}', digest=digest
```

Comment:
This change was necessary to deal with a case where `error.path` contains the index of the affected value in a list of values (e.g., `deque(['manifests', 0, 'mediaType'])` is read as "the 0(1st)th entry in a list of manifests contains invalid mediaType").

Reply:
I did not understand this comment. Can you give a specific example?

Reply:
Before:
```
Started background task /pulp/api/v3/tasks/7528dc91-9719-44fe-88e9-3810f53cb444/
................Error: Task /pulp/api/v3/tasks/7528dc91-9719-44fe-88e9-3810f53cb444/ failed: 'sequence item 1: expected str instance, int found'
```

After:
```
Started background task /pulp/api/v3/tasks/55a60363-d495-4594-b80a-3100d56b8439/
....Error: Task /pulp/api/v3/tasks/55a60363-d495-4594-b80a-3100d56b8439/ failed: '{'errors': [{'code': ErrorDetail(string='MANIFEST_INVALID', code='parse_error'), 'message': ErrorDetail(string="manifests.0.mediaType: 'application/vnd.docker.distribution.manifest.v2+json' is not one of ['application/vnd.oci.image.manifest.v1+json', 'application/vnd.oci.image.index.v1+json']", code='parse_error'), 'detail': {'digest': ErrorDetail(string='9a0a39570a12cf465209db9ab02812b02b87320bfcc1cee58978d1129191d0c8', code='parse_error')}}]}'
```